components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/utils/LoggerUtils.java

```diff
@@ -144,6 +144,17 @@ public static void triggerDiagnosticLogEvent(String componentId, Map<String, Obj
             String flowId = MDC.get(FLOW_ID_MDC);
             DiagnosticLog diagnosticLog = new DiagnosticLog(id, recordedAt, requestId, flowId, resultStatus,
                     resultMessage, actionId, componentId, input, configurations);
+            /* As the Console and MyAccount application are managed by the identity server, the diagnostic logs are not
+            required to be emitted. */
+            if (diagnosticLog.getInput() != null) {
+                String clientID = (String) diagnosticLog.getInput().get(LogConstants.InputKeys.CLIENT_ID);
+                if (clientID == null) {
+                    clientID = (String) diagnosticLog.getInput().get("client_id");
+                }
+                if ("CONSOLE".equals(clientID) || "MY_ACCOUNT".equals(clientID)) {
```

Comment:
Can we please move them to the constants?

Reply:
Fixed by - https://github.com/wso2/carbon-identity-framework/pull/5690/commits/70c201576ee3aa00e9ad8ec1344ab429fdbb0b9a